api/src/main/java/ai/djl/modality/Classifications.java

```diff
@@ -130,6 +138,12 @@ public Classifications(List<String> classNames, NDArray probabilities) {
         return null;
     }
 
+    /** {@inheritDoc} */
```

Comment:
This is a violation against separation of concern (SoC) principle.

if there are will be other forms of serialisation should every class implements its own way of toJson, ToSoap, toProtoBuf, toJMS ? 
I would say a class should not know about how it can be serialized